examples/two_neuron_example.py

```diff
@@ -24,57 +27,53 @@
 # === Define parameters ========================================================
 
 cell_params = {
-    "v_rest": -70.0, # (mV)
-    "v_reset": -70.0, # (mV)
-    "cm": 0.250, # (nF)
-    "tau_m": 10, # (ms)
-    "tau_refrac": 2, # (ms)
-    "tau_syn_E": 2, # (ms)
-    "tau_syn_I": 2, # (ms)
-    "v_thresh": -55.0, # (mV)
-    "i_offset": 0, # (nA)
+    "v_rest": -70.0,  # (mV)
+    "v_reset": -70.0,  # (mV)
+    "cm": 0.250,  # (nF)
+    "tau_m": 10,  # (ms)
+    "tau_refrac": 2,  # (ms)
+    "tau_syn_E": 2,  # (ms)
+    "tau_syn_I": 2,  # (ms)
+    "v_thresh": -55.0,  # (mV)
+    "i_offset": 0,  # (nA)
 }
 
 # === Build the network ========================================================
 
-sim.setup(timestep=0.1) # (ms)
+sim.setup(timestep=0.1)  # (ms)
```

Comment:
I changed the timestep in the previous example because for that example pyNN.brian2 did not give correct results with timestep=0.1 due to the Poisson noise mechanism. Here there is no Poisson mechanism, so timestep=0.1 gives accurate results.

Reply:
Ohk understood. So it's intentional. 

In case if I am in similar situation, I would try to check both 0.1 as well as 0.01